src/base/static/components/app.js

```diff
@@ -108,7 +110,11 @@ class App extends Component {
             include_submissions: true,
             include_tags: true,
           },
-          includePrivate: this.props.hasAdminAbilities(config.slug),
+          includePrivate: this.props.hasGroupAbilitiesInDatasets({
```

Comment:
We no longer make the assumption that only `administrators` can access private Places. Now anyone in a group with `can_access_protected` permissions can.